src/aiida_quantumespresso/parsers/xspectra.py

```diff
@@ -87,6 +87,28 @@ def parse(self, **kwargs):
         xy_data.set_y(y_arrays, y_names, y_units)
 
         parsed_data, logs = parse_stdout_xspectra(filecontent=out_file, codename='XSpectra')
+
+        # Parse some additional info which the stdout does not reliably report
+        input_params = self.node.get_incoming().get_node_by_label('parameters').get_dict()
+
+        epsilon_vector = [
+            float(input_params['INPUT_XSPECTRA'][f'xepsilon({n})']) for n in [1, 2, 3]
+        ]
+        parsed_data['epsilon_vector'] = epsilon_vector
+
+        if 'xcoordcrys' in input_params['INPUT_XSPECTRA']:
+            if input_params['INPUT_XSPECTRA']['xcoordcrys']:
+                parsed_data['vector_coord_system'] = 'crystal'
+            else:
+                parsed_data['vector_coord_system'] = 'cartesian'
+        else:
+            parsed_data['vector_coord_system'] = 'crystal'
+
+        if 'xonly_plot' in input_params['INPUT_XSPECTRA']:
+            parsed_data['only_plot_spectrum'] = input_params['INPUT_XSPECTRA']['xonly_plot']
+        else:
+            parsed_data['only_plot_spectrum'] = False
```

Comment:
This common pattern can be simplified with the `get(key, default)` paradigm:
```suggestion
        parsed_data['only_plot_spectrum'] = parameters.get('xonly_plot', False)
```